activerecord/test/models/hamster.rb

```diff
@@ -0,0 +1,4 @@
+class Hamster < ActiveRecord::Base
```

Comment:
Would it be possible to avoid creating a new model ? There's already a bunch of models in the suite. You can use an existing one and create a class inline like:

~~~ruby
klass = Class.new(Book) do
  validates :author, presence: true
end
~~~